pullRequestHandler_test.go

```diff
@@ -95,3 +95,34 @@ func Test_hasNoDcoLabel(t *testing.T) {
 		})
 	}
 }
+func Test_firstTimeContributor(t *testing.T) {
+	var authorLabel = []struct {
+		label        string
+		expectedBool bool
+	}{
+		{
+			label:        "NONE",
+			expectedBool: true,
+		},
+		{
+			label:        "CONTRIBUTOR",
+			expectedBool: false,
+		},
+		{
+			label:        "CONTRIBUTOR",
+			expectedBool: false,
+		},
+		{
+			label:        "NONE",
+			expectedBool: true,
+		},
+	}
+	for _, test := range authorLabel {
+		t.Run(test.label, func(t *testing.T) {
+			isFirstTime := firstTimeContributor(test.label)
+			if isFirstTime != test.expectedBool {
+				t.Errorf("First time contributor - %s - wanted %t, found %t", test.label, test.expectedBool, isFirstTime)
```

Comment:
Good work with this. Since you're going to make a change anyway let's set "want" "got" instead of wanted/found for consistency.